index.js

```diff
@@ -89,6 +104,7 @@
           }
         }.bind(this), RATE_LIMIT);
       }.bind(this);
+      document.addEventListener(visibilityChange, this._rcv_fn, false);
```

Comment:
no need to add the third argument *unless* it's `true` (which is an absolutely useless flag anyway).